releasenotes/notes/make_user_user_jmx_command_instead_of_check-e374f41fc5cdcddd.yaml

```diff
@@ -0,0 +1,4 @@
+deprecations:
+  - |
+    The parameter ``jmx`` is deprecated on the command ``check``, it is replaced by the command ``jmx``.
```

Comment:
I feel like this is more of a bugfix as it never worked and we already had the `jmx` command.